gslib/commands/cp.py

```diff
@@ -373,43 +364,41 @@
   or service problems.
 
   Streaming transfers using the XML API do not support resumable
-  uploads/downloads. If you have a large amount of data to upload (say, more
-  than 100 MiB) it is recommended that you write the data to a local file and
-  then copy that file to the cloud rather than streaming it (and similarly for
-  large downloads).
+  uploads or downloads. If you have a large amount of data to upload or download,
+  over 100 MiB for example, we recommend that you write the data to a file and
```

Comment:
It feels like including "local" in "local file" probably is important here.

Reply:
Done